3.preprocess-features/README.md

```diff
@@ -0,0 +1,54 @@
+# 3. Preprocess Features
+
+In this module, we present our pipeline for preprocessing features.
+
+### Feature Preprocessing
+
+We use [PyCytominer](https://github.com/cytomining/pycytominer) to compile single-cell features.
+
+We use [sklearn.preprocessing.StandardScaler](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html) to derive a normalizion scaler from all negative control features.
+`StandardScaler()` standardizes features by removing the mean and scaling to unit variance.
+
+[Caicedo et al, 2017](https://www.nature.com/articles/nmeth.4397) explain why the negative control features are a good normalization population for our use case:
+> When choosing the normalizing population, we suggest the use of control samples (assuming that they are present in sufficient quantity), because the presence of dramatic phenotypes may confound results. This procedure is good practice regardless of the normalization being performed within plates or across the screen.
+
+The `get_negative_control_index_df` function inside [preprocess-features-utils.py](preprocess-features-utils.py) creates a DeepProfiler-style `index.csv` file (saved in [norm_pop_index.csv](norm_pop_index.csv)) with only the negative control wells.
+We then use pycytominer to compile the features from these negative control wells and derive and `StandardScaler` with only the negative control features (saved in [negative_control_scaler.save](negative_control_scaler.save)).
+
+After deriving a normalization scaler, we load features by plate and apply the normalization scaler to these plate features.
```

Comment:
I'll take a closer look in the actual code, but I want to confirm that you are creating a normalization scaler _per plate_. A single multi-plate scaler is likely to induce pretty awful batch effects.

Reply:
We were not doing normalization per plate but we now are!